NutritionManagement/src/main/Day.java

```diff
@@ -2,20 +2,29 @@
 
 import java.time.LocalDate;
 import java.util.*;
+import java.util.stream.Stream;
 
 public class Day {
 	public Day(String dateString) {
-		//WIP
-		try{
-			this.date=LocalDate.parse(dateString);
-			this.foodPortions=new ArrayList<>();
-		}
-		catch(Exception e){
-			
+		// WIP
+		try {
+			this.date = LocalDate.parse(dateString);
```

Comment:
when the date string parsing fails, the exception is merely caught without providing further feedback. Recommended to give a clear prompt when the date parsing fails, or to notify the caller by throwing an exception that the date format is incorrect.

An example attmpt:
```
public Day(String dateString) {
  try {
    this.date = LocalDate.parse(dateString);
  } catch (DateTimeParseException e) {
    throw new IllegalArgumentException("Date format is incorrect, please use the correct format: YYYY-MM-DD");
  }
  this.foodPortions = new ArrayList<>();
}
```

Reply:
This part is still WIP. Noted and will update in next MR